edb/testbase/server.py

```diff
@@ -1559,6 +1561,22 @@ async def check_branching(self, include_data=False, *, check_method):
         oldcon = self.con
         self.__class__.con = con2
         try:
+            # We cannot compare the SDL text of the new branch schema to the
+            # original because the order in which it renders all the
+            # components is not guaranteed. Instead we will use migrations to
+            # compare the new branch schema to the original. We expect there
+            # to be no difference and therefore a new migration to the
+            # original schema should have the "complete" status right away.
+            await self.con.execute(f'start migration to {{ {orig_schema} }}')
+            mig_status = json.loads(
+                await self.con.query_single_json(
+                    'describe current migration as json'
+                )
+            )
+            self.assertTrue(mig_status.get('complete'))
+            # cancel the migration
+            await self.con.execute('rollback')
```

Comment:
Do `abort migration` instead of `rollback`. `rollback` will rollback an enclosing connection too; that doesn't really matter here, but it's cleaner

Reply:
Fixed